charts/camunda-platform/values.yaml

```diff
@@ -1754,7 +1754,26 @@ connectors:
   # Switch for inbound mode (e.g., for webhook or polling)
   inbound:
     # Acceptable values: disabled, credentials, or oauth
-    mode: disabled
+    mode: oauth
+
+    # Operate client configurations
+    operateClient:
+      # Specify endpoint if required
+      endpoint:
+
+      # When inbound.mode: credentials
+      username: demo
+      password: demo
+
+      # When inbound.mode: oauth
+      operateOAuthClientId: connectors
+      operateOAuthClientSecret: c0nn3ct0rsAr3Aw3s0me
```

Comment:
The secrets should not be used directly in the values file but ref to an existing secret.

Search for `existingSecret` in the repo to find implementation examples.

Reply:
In fact, I think the whole logic of `credentials` or `oauth` should be moved to the secret file.